lib/presentation/crowd_action/crowd_action_details.dart

```diff
@@ -1,61 +1,313 @@
-// TODO: Implement class once GraphQL and Models are configured properly
-// import 'package:collaction_app/domain/crowdaction/crowdaction.dart';
-// import 'package:flutter/material.dart';
+import 'package:collaction_app/presentation/shared_widgets/rectangle_button.dart';
+import 'package:collaction_app/presentation/themes/constants.dart';
+import 'package:flutter/material.dart';
+import 'package:url_launcher/url_launcher.dart';
 
-// /// Route displaying more information of a specific selected Collaction
-// class CrowdActionDetailsPage extends StatefulWidget {
-//   final CrowdAction model;
+import '../../domain/crowdaction/crowdaction.dart';
+import '../shared_widgets/custom_appbar.dart';
 
-//   const CrowdActionDetailsPage({Key? key, required this.model})
-//       : super(key: key);
+class CrowdActionDetailsPage extends StatefulWidget {
```

Comment:
I would prefer singular for this name (`CrowdActionDetailPage`)

Reply:
But the correct english for it is plural, it is not one detail, it is multiple details about the CrowdAction.

A name is a detail.
A goal is a detail.
A description is a detail.

Together they are details.

Reply:
Ok yeah... makes sense.  
(I was thinking about the master/detail page hierarchy.)